app/addons/replication/__tests__/helpers-tests.js

```diff
@@ -0,0 +1,34 @@
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
```

Comment:
I have found <file>.test.js to work quite well, but I have seen the above pattern quite often.